themes/default/theme.php

```diff
@@ -6,12 +6,17 @@ abstract class Theme implements ThemeInterface {
     protected $vars = array();
     protected $pageTitle = null;
     protected $register = array('script' => array(), 'style' => array());
+    protected $rendered = "";
 
     public function __construct(&$main, &$twig, $vars) { }
 
     public function render() {
         $this->vars['register'] = $this->register;
-        echo $this->document->render($this->vars);
+        $this->rendered = $this->document->render($this->vars);
```

Comment:
Why not just return the rendered string? Seems like everywhere this is called it's followed by getRendered() exactly once :thinking:

Reply:
I had plans to do some post processing content adjustment in a plugin for it, but probably won't commit this as its a little outside the scope of the project.

Reply:
Since you've mentioned you're not likely to submit your other changes, I think I'd prefer what Chris suggested